packages/front-end/components/Archetype/ArchetypeResults.tsx

```diff
@@ -428,26 +433,30 @@ const ArchetypeResults: FC<{
                 )}
                 <td className={styles.showOnHover}>
                   <MoreMenu>
-                    <button
-                      className="dropdown-item"
-                      onClick={() => {
-                        setEditArchetype(archetype);
-                      }}
-                    >
-                      Edit
-                    </button>
-                    <DeleteButton
-                      className="dropdown-item"
-                      displayName="Archetype"
-                      text="Delete"
-                      useIcon={false}
-                      onClick={async () => {
-                        await apiCall(`/archetype/${archetype.id}`, {
-                          method: "DELETE",
-                        });
-                        onChange();
-                      }}
-                    />
+                    {canEdit ? (
```

Comment:
In other areas of the app, we don't show edit or delete options if we know the user doesn't have permission - this update makes this table behave more consistently with other areas of the app.